TwitchLib.Client/Events/OnRaidedChannelIsMatureAudienceArgs.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+
+namespace TwitchLib.Client.Events
+{
+    /// <summary>
+    /// Args representing a NOTICE telling the client the newly raided channel is mature audience only.
+    /// Implements the <see cref="System.EventArgs" />
+    /// </summary>
+    /// <seealso cref="System.EventArgs" />
+    /// <inheritdoc />
+    public class OnRaidedChannelIsMatureAudienceArgs : EventArgs
+    {
+        /// <summary>
+        /// Property representing channel bot is connected to.
+        /// </summary>
+        public string Channel;
```

Comment:
Just fields? No auto-property?

Reply:
Other events use fields too. It doesn't really matter. This is one of the most "OOP" codebases I've worked with in a while and it definitely has no business being this abstraction-heavy (and have 51 event args classes).

Reply:
maybe we should create one new event\* and use it wherever it makes sense instead of creating more useless events


\* event
```cs
public class NoticeEventArgs : EventArgs
{
// todo ctor or publish setter
      public string Channel { get; }
      public string Message { get; }
}
```
this event can replace cca 11 events

Reply:
Thanks, good idea.

Reply:
But we can do it in separate PR. Either me or AoshiW so it's not again on you.

Reply:
Sure, It's worth doing it separately as I wanted to take another look at event args and extend overall message-to-handler test coverage for them, current tests are brittle and do not guard against mistakes (we can build a map of irc payload -> expected event args object to ensure exact behavior).